region/maridia/inner-green/East Sand Hall.json

```diff
@@ -985,6 +1069,111 @@
         }
       }
     },
+    {
+      "link": [2, 2],
+      "name": "Suitless Evir Farm (1 Evir, Grapple)",
+      "requires": [
+        {"resetRoom": {"nodes": [2]}},
+        "canDodgeWhileShooting",
+        "canSuitlessMaridia",
+        "HiJump",
+        "Grapple",
+        {"or": [
+          {"and": [
+            "Plasma",
+            {"cycleFrames": 240}
+          ]},
+          {"and": [
+            "Wave",
+            "canFarmWhileShooting",
+            {"cycleFrames": 420}
+          ]}
+        ]}
+      ],
+      "farmCycleDrops": [{"enemy": "Evir", "count": 1}],
+      "note": [
+        "If Plasma is available, quickly fire two shots when entering to kill the Evir;",
+        "it can help to crouch for the second shot.",
+        "If only Wave is available, quickly fire three shots,",
+        "then jump out exactly one tile away from the ledge",
+        "and fire three more shots angled-down to kill the Evir.",
+        "In any case, use Grapple to collect the drop from the pillar without needing to drop down."
+      ],
+      "devNote": [
+        "If Gravity is available, then other farm strats can be used instead."
+      ]
+    },
+    {
+      "link": [2, 2],
+      "name": "Evir Farm (1 Evir, Gravity, Screw Attack)",
+      "requires": [
+        {"resetRoom": {"nodes": [2]}},
+        "Gravity",
+        "ScrewAttack",
+        "canCarefulJump",
+        {"cycleFrames": 290},
+        {"or": [
+          "canWalljump",
```

Comment:
I'm not so sure Medium can handle this walljump.  Could you think about adding a spacejump option?
With HiJump, you would probably eventually happen to get a good jump.

Existing strats have canPlayInSand on these movement options.

Reply:
Sounds good. Added a Space Jump option and put canPlayInSand on the others. The Gravity+HiJump option feels close to the line between Medium and Hard. Makes sense not to expect more in the farm strat than is expected in the room traversal strat, probably safest to keep it in Hard for now.